src/ui/control/geolocate_control.js

```diff
@@ -326,9 +350,11 @@ class GeolocateControl extends Evented {
         // when showUserLocation is enabled, keep the Geolocate button disabled until the device location marker is setup on the map
         if (this.options.showUserLocation) {
             this._dotElement = DOM.create('div', 'mapboxgl-user-location-dot');
-
             this._userLocationDotMarker = new Marker(this._dotElement);
 
+            this._circleElement = DOM.create('div', 'mapboxgl-user-location-accuracy-circle');
+            this._accuracyCircleMarker = new Marker(this._circleElement);
```

Comment:
Could you set `pitchAlignment: 'map'` so that when the map is pitched the accuracy circle remains flat on the map?

Reply:
😮

Reply:
Well that is pretty cool!

![image](https://user-images.githubusercontent.com/10985/72480289-f9114e00-37c4-11ea-8117-06cb261d68e6.png)

Reply:
Yeah I was trying to make the same change to the dot in https://github.com/mapbox/mapbox-gl-js/pull/8929/files#diff-8c498cf163efa60ad84199c531ab7c5aR368-R372 but if you want you could do the same to the dot here.